rtlib/include/rt/LandmarkIO.hpp

```diff
@@ -5,6 +5,7 @@
 #include "rt/ImageTypes.hpp"
 #include "rt/LandmarkRegistration.hpp"
 
+namespace fs = boost::filesystem;
```

Comment:
Only do namespace aliases in cpp's. Otherwise they get inherited by everything that includes this header.